server/controllers/student/getStudents.js

```diff
@@ -0,0 +1,24 @@
+const { getStudentsInformation } = require('../../database/queries/getStudents');
+
+const getStudents = (req, res) => {
+  getStudentsInformation()
+    .then((students) => {
+      if (!students.rowCount) {
+        res.send({
+          error: 'No students in the database',
```

Comment:
i think it's not an error. ```suggestion
 res.send({ data: [] }),```